cognite/client/cognite_client.py

```diff
@@ -0,0 +1,194 @@
+import logging
+import os
+from typing import Any, Dict
+
+from cognite_logger import cognite_logger
+
+from cognite.client._api_client import APIClient
+from cognite.client.experimental import ExperimentalClient
+from cognite.client.stable.assets import AssetsClient
+from cognite.client.stable.datapoints import DatapointsClient
+from cognite.client.stable.events import EventsClient
+from cognite.client.stable.files import FilesClient
+from cognite.client.stable.login import LoginClient
+from cognite.client.stable.raw import RawClient
+from cognite.client.stable.tagmatching import TagMatchingClient
+from cognite.client.stable.time_series import TimeSeriesClient
+
+DEFAULT_BASE_URL = "https://api.cognitedata.com"
+DEFAULT_NUM_OF_RETRIES = 5
+DEFAULT_NUM_OF_WORKERS = 10
+DEFAULT_LOG_LEVEL = "DEBUG"
```

Comment:
That it can spam quite a lot when fetching dps in parallell and i have found it annoying. Should we set it to INFO instead?

Reply:
The reason for setting default log level to DEBUG is because you find it annoying? 😅 Wouldn't it be natural for users to have default log level at WARNING? When you use a library you generally only want warnings and errors - which I believe is the norm?

Reply:
Ah, okey, sorry - just assumed this was displayed somehow to the user.